test/uk/gov/hmrc/cgtpropertydisposals/connectors/dms/GFormConnectorSpec.scala

```diff
@@ -93,7 +93,7 @@ class GFormConnectorSpec extends WordSpec with Matchers with MockFactory with Ht
       val dms = DmsSubmissionPayload(
         B64Html("html"),
         List(FileAttachment("key", "filename", Some("application/pdf"), Seq(ByteString("data")))),
-        DmsMetadata("id", "cId", "ct", "ba")
+        DmsMetadata("id", "cId", "ct", "ba", backscan = true)
```

Comment:
would be good to create a generator for `DmsMetadata`

Reply:
I'll do that quickly now

Reply:
changed it to use generators now - turns out there was already one for it